.github/actions/composite/announceFailedWorkflowInSlack/action.yml

```diff
@@ -16,13 +22,14 @@ runs:
         fields: workflow, repo
         custom_payload: |
           {
-            channel: '#announce',
+            channel: '${{ inputs.CHANNEL }}',
```

Comment:
It seems like this is how its used in other places for deployer channel https://github.com/Expensify/App/blob/3d79c9c54ea1cd616adf4af3e898d55302d614a4/.github/workflows/platformDeploy.yml#L399